scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/Configuration.java

```diff
@@ -98,7 +97,9 @@
     private String javaHome = "";
     @Parameter(names = {USE_IP_ADDRESS}, arity = 1, description = "If true, the framework will resolve the local ip address. If false, it uses the hostname.")
     private Boolean isUseIpAddress = false;
```

Comment:
This is not exactly about this PR, however should this member be named `useIpAddress` with getter `isUseIpAddress()` and setter `setUseIpAddress()`? This is close to getter/setters naming convention for booleans